app/views/users/_proposal.html.erb

```diff
@@ -0,0 +1,28 @@
+<tr id="proposal_<%= proposal.id %>">
+  <td>
+    <%= link_to proposal.title, proposal, proposal.retired? ? { class: 'retired' } : {} %>
```

Comment:
I like that `? { class: 'retired' } : {}` :) I think I have added some `class=""` around :( couldn't figure out this was the way 👏